src/styles/stop-watch.scss

```diff
@@ -0,0 +1,66 @@
+.stop-watch {
+  display: flex;
+  justify-content: center;
+  &__container {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 500px;
+
+    &:hover > * {
```

Comment:
don't you * selector. You can use classes

`
&:hover > second-hand-class,
&:hover > minute-hand-class {
`